cypress/e2e/testtab.cy.js

```diff
@@ -171,11 +173,19 @@ function copyTheReportsToTestTab() {
   cy.wait(100);
   cy.checkFileTreeLength(2);
   cy.wait(100);
-  cy.get('[data-cy-debug-tree="root"] > app-tree-item').eq(0).find('.item-name').eq(0).click();
+  if(testSequence) {
+    cy.get('[data-cy-debug-tree="root"] > app-tree-item').eq(0).find('.item-name').eq(0).click();
+  } else {
+    cy.get('[data-cy-debug-tree="root"] > app-tree-item').contains('Simple report').click()
+  }
```

Comment:
```suggestion
cy.get('[data-cy-debug-tree="root"] > app-tree-item').contains('Simple report').click()
```
Since this function is only being called from beforeEach and never changes, why not changes it to the new line permanently instead? 
Is there a reason we would need to test the sequence?

Reply:
Gedaan. Ik heb de oude regel (die bij de then hoorde) laten staan als comment. Die regel willen we terug als de volgorde in de debug tree klopt, dus als issue https://github.com/wearefrank/ladybug-frontend/issues/439 gefixed is.